src/router/index.ts

```diff
@@ -1,13 +1,14 @@
 import { createRouter, createWebHistory } from 'vue-router'
 import HomeView from '../views/HomeView.vue'
+import Signup from '../views/Signup.vue'
 
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
-      path: '/',
-      name: 'home',
-      component: HomeView
+      path: '/signin',
+      name: 'signin',
+      component: Signup
```

Comment:
maybe we could set signIn path as default when opening the website? at least if user isn't loggedIn